music21/sorting.py

```diff
@@ -105,7 +105,7 @@ def __eq__(self, other):
             elif self.atEnd == 1:
                 return True
             else:
-                return self.offset == other
+                return (self.offset == other)
```

Comment:
revert since it doesn't change output and we're trying to reduce extraneous parentheses.